### PR TITLE
Admin rent control

### PR DESCRIPTION
**Pull request by tmassalski**

### CODE REVIEW

**Review by kacper-cyra — DISMISSED** [review]

File: src/App.jsx [hunk 1]
```diff
@@ -28,6 +30,12 @@ export const App = () => {
             <Route path="/reservations">
               <ReservationPage />
             </Route>
+            <Route path="/rentBikes">
```
Comment (MarKust71) [inline src/App.jsx:33]: it is safer not to use camel/pascal case in routes (for HTTP requests), I think. i.e. 'rent-bikes' would be better

File: src/components/return/ReturnTable.js [hunk 2]
```diff
@@ -0,0 +1,228 @@
...
+  }
+
+  return 0;
+}
+
+function getComparator(order, orderBy) {
+  return order === 'desc'
+    ? (a, b) => descendingComparator(a, b, orderBy)
+    : (a, b) => -descendingComparator(a, b, orderBy);
+}
```
Comment (MarKust71) [inline src/components/return/ReturnTable.js:32]: +1/-1 changes here, only. Maybe this way would be easier to read and understand?
`return (a,b) => descendingComparator(a, b, orderBy) * (order === 'desc' ? 1 : -1)`

File: src/pages/reservationPage/ReservationPage.jsx [hunk 3]
```diff
@@ -1,9 +1,12 @@
 import React, { useState, useEffect } from 'react';
 import { Container, Grid, Paper } from '@material-ui/core';
+import { Alert } from '@material-ui/lab';
 import DateFnsAdapter from '@material-ui/lab/AdapterDateFns';
 import LocalizationProvider from '@material-ui/lab/LocalizationProvider';
-import CircularProgress from '@material-ui/core/CircularProgress';
+import LinearProgress from '@material-ui/core/LinearProgress';
 import Typography from '@material-ui/core/Typography';
+import Snackbar from '@material-ui/core/Snackbar';
+import Slide from '@material-ui/core/Slide';
```
Comment (MarKust71) [inline src/pages/reservationPage/ReservationPage.jsx:9]: The above 4 lines can be reduced into one:
`import { LinearProgress, Typography, Snackbar, Slide} from '@material-ui/core'`

**Review by MarKust71 — APPROVED** [review]
I still see the tendency to build components that contain more than 150 lines. It must be the way to atomize it, divide it into more files with elements to be imported. Compare code refactored by Mikołaj.

**Pull request merged**